repos/spack_repo/builtin/packages/sherpa/package.py

```diff
@@ -37,6 +38,8 @@ class Sherpa(CMakePackage, AutotoolsPackage):
         conditional("cmake", when="@3:"), conditional("autotools", when="@:2"), default="cmake"
     )
 
+    conflicts("%gcc@15:", when="@:3.0.2")
```

Comment:
Ranges inclusive in spack.
```suggestion
    conflicts("%gcc@15:", when="@:3.0.1")
```